DESIGN.md

```diff
@@ -0,0 +1,58 @@
+# Module Design
+
+## Classes
+
+There are 4 main public classes and other can be added if they need to be managed separately than the existing classes:
```

Comment:
Use words like `four` instead of numerals.

Reply:
This change was made.